src/IIIFPresentation/Models/Database/Collections/Manifest.cs

```diff
@@ -47,6 +47,11 @@ public class Manifest : IHierarchyResource
     public List<CanvasPainting>? CanvasPaintings { get; set; }
     
     public List<Batch>? Batches { get; set; }
+    
+    /// <summary>
+    /// Whether the manifest has been ingested with assets at some point
+    /// </summary>
+    public bool Ingested { get; set; }
```

Comment:
Would there be any use in this being a `DateTime? LastProcessed` or `DateTime? LastFinished` or something? Would provide both an easy way of identifying both whether it was and when it was.

Alternatively a `bool?` as some Manifest will never Ingest with assets (ie if they've only ever been vanilla IIIF). With this `false` means "not done yet" but also "won't ever be done"